KB - Save and Load the Splitter State (#643)
telerik/blazor-docs: components/splitter/state.md

}
````

> >tip You can use the [splitter events]({%slug splitter-events%}) to save its state for your users. You can then set it every time the page is reloaded. Read more in this knowledge base article - [Save and Load the Splitter State]({%slug splitter-kb-save-and-load-state%}).

Reviewer: ```suggestion
>tip You can use the [Splitter events]({%slug splitter-events%}) to save the component state and restore it on page reload. Learn more at [Save and Load the Splitter State]({%slug splitter-kb-save-and-load-state%}).
```
